Add Guides sand SigV4 support guide (#4997)
opensearch-project/documentation-website: _benchmark/guides/index.md

---
layout: default

> title: Guides

Reviewer: ```suggestion
title: Use cases
```

nav_order: 70
has_children: true
---

> # Guides for specific use cases

Reviewer: ```suggestion
# OpenSearch Benchmark use cases
```

> Here are a list of guides for specific OpenSearch Benchmark use cases.

Reviewer: ```suggestion
The following section provides guides for specific use cases of OpenSearch Benchmark. To get started, select from the following guides:
```
